experimental/benchmarks/acmeair/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+MAINTAINER Gareth Ellis <gareth.ellis@uk.ibm.com>
+RUN apt-get update && apt-get install git openjdk-7-jdk mongodb zip unzip numactl bc wget -y
+RUN mkdir /node 
+RUN date
+WORKDIR /node
+RUN mkdir Jmeter node_undertest node_baseline
+RUN git clone http://github.com/gareth-ellis/benchmarking 
+RUN bash /node/benchmarking/experimental/benchmarks/acmeair/setupJmeter.sh
+RUN numactl --hardware
+COPY node_undertest /node/node_undertest
+COPY node_baseline /node/node_baseline
+RUN /node/benchmarking/experimental/benchmarks/acmeair/run_acmeair_docker.sh
```

Comment:
`node_undertest` and `node_baseline` could be moved into volumes at runtime, and `run_acmeair_docker` could be moved to `CMD`. This would let this group publish that docker image to the docker registry so that it could be consumed.

Running it would look something like:

`docker run -it -v node_undertest:/node/node_undertest -v node_baseline:/node/node_baseline nodejs/acmeair:latest`